components/charts/CountryBlurb.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { formatDecimal, formatKm } from '../../lib/utils/format'
+
+export default function Blurb ({
+  users,
+  km_roads_add,
+  buildings_add,
+  poi_add,
+  km_waterways_add,
+  km_coastlines_add,
+  km_coastlines_mod,
+  country
+}) {
+  return <h2 className='header--small width--shortened list--block'>
+    <mark>{users.length}</mark> mappers in <mark>{country}</mark> are mapping <mark>{formatKm(km_roads_add)}</mark> of roads, <mark>{formatDecimal(buildings_add)}</mark> buildings, <mark>{formatDecimal(poi_add)}</mark> Points of Interest, <mark>{formatKm(km_coastlines_add + km_coastlines_mod)}</mark> of coastlines, and <mark>{formatKm(km_waterways_add)}</mark> of waterways.
+  </h2>
+}
```

Comment:
@kamicut Is this a placeholder for when we get this data? I don't see a blurb when I run this locally.

Reply:
Yes correct, we don't have this data in OSMESA yet.

Reply:
👍 Just making sure. Can we make a note in https://github.com/developmentseed/scoreboard/issues/223 that we'll be coming back to this?